components/org.wso2.carbon.identity.api.server.branding.preference.management/org.wso2.carbon.identity.api.server.branding.preference.management.v1/src/main/java/org/wso2/carbon/identity/api/server/branding/preference/management/v1/core/BrandingPreferenceManagementService.java

```diff
@@ -70,7 +71,7 @@
 public class BrandingPreferenceManagementService {
 
     private static final Log log = LogFactory.getLog(BrandingPreferenceManagementService.class);
-    //TODO: Improve API to manage application level & language level theming resources in addition to the tenant level.
+    //TODO: Improve API to manage custom level & language level theming resources in addition to the tenant level.
```

Comment:
remove the todo since this is done

Reply:
Resolved with https://github.com/wso2/identity-api-server/pull/454/commits/da5ef354ea314adb7ff5c80500bfa4de0622cd50